src/common/components/card/LargeCollectionCard.tsx

```diff
@@ -16,7 +16,7 @@ function LargeCollectionCard({
     <Card id={id} className={styles.largeCollectionCard}>
       <Card.Content className={styles.text}>
         <Card.Title as="h3" variant="h2" title={title} href={href} />
-        <Card.InfoLines as="p" variant="body-xl" infoLines={infoLines} />
+        <Card.InfoLines as="p" variant="body-l" infoLines={infoLines} />
```

Comment:
Variant `body-l` should be mapped to the `p` element by default:

https://github.com/City-of-Helsinki/liikunta-helsinki/blob/main/src/common/components/text/Text.tsx#L36